### DISCUSSION TITLE
After new update, I get error: `durin` should be `during`

### DISCUSSION THREAD

**Discussion by azzamsa** [crate-ci/typos#977] [Q&A] [answered]
Hi.

I use [Durin](https://lotr.fandom.com/wiki/Durin_I) as a username in many of my projects. Now after the new update, I get ```error: `durin` should be `during` ```.

Should I abandon `durin` to avoid creating `typos.toml` in every project I have? Or is there any alternative?

Thanks! 🍨

**Comment by epage** [comment, marked as answer]
How does your username show up in every project that they require allowing in `typos.toml`.

Names will always be a difficult aspect to deal with.  We even use one name issue in the [docs as an example](https://github.com/crate-ci/typos?tab=readme-ov-file#false-positives).  I feel like the only way to deal with this is with `typos.toml` files.

**Reply by azzamsa** [reply]
> I feel like the only way to deal with this is with `typos.toml` files.

Oh, I see. Thank you!
